src/test/groovy/ee/tuleva/onboarding/payment/PaymentFixture.groovy

```diff
@@ -27,10 +31,34 @@ class PaymentFixture {
   static PaymentData aPaymentDataForMemberPayment = new PaymentData(sampleUserNonMember.personalCode, aPaymentAmount, aPaymentCurrency, PaymentType.MEMBER_FEE, PaymentChannel.TULUNDUSUHISTU)
   static aPaymentCreationTime = TestClockHolder.now - Duration.ofDays(1)
 
+  static MontonioOrder aMontonioOrder = MontonioOrder.builder()
+      .accessKey("testAccessKey")
+      .merchantReference("testMerchantReference")
+      .returnUrl("http://return.url")
+      .notificationUrl("http://notification.url")
+      .grandTotal(BigDecimal.valueOf(100.00))
```

Comment:
100.0 is BigDecimal in groovy, by default. So you can just write `.grandTotal(100.0)`